nb_train.py

```diff
@@ -0,0 +1,90 @@
+import pandas as pd
+import string
+import re
+import nltk
+import random
+import pickle
+from sklearn.model_selection import train_test_split
+from nltk.corpus import stopwords
+from zipfile import ZipFile
+
+with ZipFile('imdb ratings.zip') as zf:
+    f = zf.open('movie.csv')
+    
+df = pd.read_csv(f)
+smalldf = df[:2000]
+smalldf = smalldf.drop_duplicates(ignore_index = True)
```

Comment:
I was reading through the steps for exploratory data analysis that others have done on Kaggle, and I saw someone dropping duplicates as part of their data cleaning. So I did too. You don't think it's necessary?

Reply:
The way to find out is to check the data set for duplicates (you could write code to do this). I doubt there would be duplicates since it is a curated data set.